vamdctap-webservice/src/main/java/org/vamdc/tapservice/QueryStoreNotification.java

```diff
@@ -0,0 +1,87 @@
+package org.vamdc.tapservice;
+
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.Response;
+
+import org.vamdc.tapservice.api.RequestInterface;
+
+public class QueryStoreNotification extends Thread{
+	static final String QUERY_STORE_URL = "http://querystore.vamdc.eu/NotificationListener?";
+	static final String VAMDC_NODE_VERSION = "12.07";
+	static final String QS_USER_AGENT = "VAMDC Query store";
+	static final String TAP_URL = "http://basecol2015.vamdc.eu/12_07/TAP/sync?";
```

Comment:
This should be moved to configuration parameters.
`org.vamdc.tapservice.util.Settings` is your friend, and you already have `baseurl` there.